website1/css/header.css

```diff
@@ -0,0 +1,52 @@
+.header{
+  display: flex;
+  justify-content: center;
+}
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.logo_company {
+  width: 151px;
+  height: 81px;
+}
+
+.name_company {
+  font-weight: 700;
+  line-height: 159.02%;
+  width: 286px;
+  height: 29px;
+}
+
+.menu {
+  display: flex;
+}
+
+.menu__link {
+  margin: 10px;
+}
+
+.language {
+  margin-left: 57px;
+  margin-right: 30px;
+  font-weight: 700;
+  line-height: 159.02%;
+  color: #FFC300;
+}
+
+.btn_call {
+  border: 2px solid #FFC300;
+  width: 190px;
+  height: 48px;
+  left: 1175px;
+  top: 17px;
+  background-color: #fff;
+}
+
+.btn_call:hover{
+  background-color: #FFC300;
+  color: #fff;
+  cursor: pointer;
+  transition: 0.3s;
```

Comment:
Move `transition` by default, because when the cursor leaves the element, animations need too.